src/amqp/queue.js

```diff
@@ -11,6 +11,9 @@ var topLog = require( "../log" )( "rabbot.topology" );
 var unhandledLog = require( "../log" )( "rabbot.unhandled" );
 var noOp = function() {};
 
+// Disable caching because of memory leak
+postal.configuration.resolverNoCache = true;
```

Comment:
Can you make this configurable (and disabled by default), so that we could submit this as a pull request upstream to the rabbot project?